package/doc/sphinx/source/index.rst

```diff
@@ -73,69 +73,6 @@ use the MDAnalysis library which may be of interest.
 
 .. _installation-instructions:
 
-Installing MDAnalysis
-=====================
```

Comment:
I think I would still leave this section for clarity, and ease of reference. However, I appreciate the two sections would be very much interlinked (since the installation instructions would just be a link to the User Guide).

Maybe a solution could be simply to change the section name above to something like "Installing and Using MDAnalysis"?